src/queryExecutionContext/EndpointComponent/OrderedDistinctEndpointComponent.ts

```diff
@@ -0,0 +1,31 @@
+import { sha1 } from "crypto-hash";
```

Comment:
@bchong95 @christopheranderson I ended up going with sha1 (160 bits) for this hash as it is basically zero bundle cost. The browser and node both have a built-in sha1 hash and the `crypto-hash` package handles loading the correct one in each environment. Happy to switch this over to murmurhash once that returns, but this seems like the simplest thing for now.

Reply:
Im good with sha1, since the hash doesn't need to be crypographically secure:

https://en.wikipedia.org/wiki/SHA-1#SHAttered_%E2%80%93_first_public_collision

Only vulnerablity I can think of is if someone using CosmosDB allows a user to insert user data and they generate a collision to make another person's document not show up in the distinct result. But as the wiki says it would like 1100 years of CPU time, which I am not too worried about.

Reply:
I'll put something on the calendar for 3119